README.md

```diff
@@ -0,0 +1,18 @@
+# Pay performance reporter
+
+We have an AWS task that runs daily to send stats to the [GOV.UK Performance platform](https://www.gov.uk/performance/govuk-pay).
```

Comment:
`and AWS ECS task` is more accurate

Reply:
Ok cool, I have no idea what I am talking about 🤷‍♂️